recipe/meta.yaml

```diff
@@ -36,15 +36,15 @@ requirements:
   run:
     - python
     - scipy >=1.5.0
-    - {{ pin_compatible('numpy', lower_bound='1.21') }}
+    - {{ pin_compatible('numpy', lower_bound='1.22.3') }}
```

Comment:
For extra context we have a custom conda_build_config.yaml that kinda overrides this. Part of the task here will be to know how that works when you need different things at build time and run time.